Add 'Bumping iOS Build Number' page (#19)
expo/fyi: bumping-ios-build-number.md

# Bumping iOS Build Number

#### 🤔 What Happened

If you're submitting your iOS app to App Store using the `eas submit` command, you may encounter the error saying you've already submitted a particular version of the app.

#### 💡 Solution

> 1. - **If you are building your app using EAS Build**

Reviewer: ```suggestion
1. - **If you are building a managed app**
```

>      Increment the value for the `expo.ios.buildNumber` key in `app.json`. Remember that this value must be a string. Optionally, commit this change. Then, build a new app archive with `eas build`. Wait for the build to finish.

Reviewer: this won't actually update the build number for a bare react-native project that runs on eas build, at the moment. it'll only work for *managed* projects.

     [<img src="./assets/bumping-ios-build-number/01-bumping-ios-build-number.png" width="800" />](./assets/bumping-ios-build-number/01-ios-build-number-code.png)

>    - **If you are building your app using Xcode**

Reviewer: ```suggestion
   - **If you are building a bare React Native app**
```
